app/controllers/report_templates_controller.rb

```diff
@@ -48,4 +54,14 @@ def action_permission
         super
     end
   end
+
+  def build_input_values_from_query(params)
+    @report_template = ReportTemplate.find(params[:id])
```

Comment:
I'm not sure why `before_action :find_report_resource` didn't do this for me. :confused: